app/views/cloud_tenant/edit.html.haml

```diff
@@ -1,4 +1,8 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "cloudTenantFormController"}
+%form#form_div{:name => "angularForm",
+                'ng-controller' => "cloudTenantFormController as vm",
```

Comment:
broken alignment here ;) .. `'` should be under `:`, and the first `=>` should be aligned with the rest..